src/components/LanguageButton/LanguageButton.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
 import { Button } from "react-native-elements";
 import PropTypes from "prop-types";
+import AsyncStorage from "@react-native-community/async-storage";
 import { styles } from "./styles";
+import { changeLocale } from "../../i18n";
 
 const LanguageButton = props => {
-  const { title, langKey, navigation } = props;
-  /* eslint-disable-next-line */
-  const onPress = key => {
-    navigation.navigate("auth");
+  const { title, langKey, navigation, screenProps, navigationRoute } = props;
+
+  const onPress = async key => {
+    changeLocale(key);
+    screenProps.onLocaleChange(key);
+    navigation.navigate(navigationRoute);
+    await AsyncStorage.setItem("lang", key);
```

Comment:
does it makes sense to put this line into the `changeLocale` function inside translator.js?

Reply:
For me it makes sense

Reply:
fixed it